front/lib/api/config.ts

```diff
@@ -121,6 +121,13 @@ const config = {
   getTextExtractionUrl: (): string => {
     return EnvironmentConfig.getEnvVariable("TEXT_EXTRACTION_URL");
   },
+  // Status page.
+  getProviderStatusPageId: (): string => {
+    return EnvironmentConfig.getEnvVariable("PROVIDER_STATUS_PAGE_ID");
+  },
+  getStatusPageApiToken: (): string => {
```

Comment:
is that key only for the provider status page ? or works also for main status page ?

Reply:
It works for both.